### PR TITLE
Keep custom logger name in EventSourcedBehavior 

### PR DESCRIPTION
**Pull request by johanandren**
Refs #28161

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]
LGTM

File: akka-actor-typed/src/main/mima-filters/2.6.1.backwards.excludes/issue-28161-logger-name.excludes [hunk 1]
```diff
@@ -0,0 +1 @@
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.typed.internal.ActorContextImpl#LoggingContext.this")
```
Comment (chbatey) [inline akka-actor-typed/src/main/mima-filters/2.6.1.backwards.excludes/issue-28161-logger-name.excludes]: not sure if mima plugin handles files without new lines at the end?

Comment (patriknw) [reply akka-actor-typed/src/main/mima-filters/2.6.1.backwards.excludes/issue-28161-logger-name.excludes]: according to contributing.md the directory name should be 2.6.0...
`<last-version>.backwards.excludes`

**Pull request merged**